ui/OptionsHandler.rb

```diff
@@ -1,21 +1,24 @@
 require_relative '../App'
+require_relative 'ExitHandler'
 
 class OptionHandler
   def initialize(app)
     @app = app
   end
 
-  def handle_option(option, _app)
+  def handle_option(option)
     case option
     when 1 then @app.list_all_books
     when 2 then @app.list_all_people
     when 3 then @app.create_person
     when 4 then @app.create_book
     when 5 then @app.create_rental
     when 6 then @app.list_rentals_from_person_id
```

Comment:
- [ ] Please make sure that you fix this error I get below when listing all rentals for a given person's `id` :pray: 
  
  ![image](https://github.com/PabloEGonz/school-library/assets/92303891/4b6dcee7-e2cd-4cd3-89be-332243538634)